src/lib/hooks/useSideMenu.js

```diff
@@ -168,6 +168,7 @@ export default (props = {}) => {
         action: () => {
           fireEvent(LOGOUT)
           AsyncStorage.clear()
+          AsyncStorage.setItem('isFirstTime', false) //dont show welcome intro next time
```

Comment:
does this flags wasn't set to false _just after_ into pages were shown ?

Reply:
i restore this flag when user logs out. (there's a .clear() before)
so we dont show the intro screens again

Reply:
the requirement was to show into **once on a browser / device**
so if user viewed intro, logged in then logged out he shouldn't see intro pages again